src/Globals.d.ts

```diff
@@ -1,2 +1,9 @@
-declare module "*.css";
-declare module "*.scss";
\ No newline at end of file
+// For SCSS modules
+declare module '*.scss' {
+    const classes: { [key: string]: string };
```

Comment:
how this declare works?

Reply:
In this declaration, you create a module for each .scss file and define an object class. It's more specific

You can check console.log(css) in index.ts in two cases for details